app/views/main/index.html.haml

```diff
@@ -1,2 +1,19 @@
-%p
-  some text
+- if !@card.blank?
+  .row
+    .col-md-6.col-md-offset-3
+      .thumbnail.card
+        %h2
+          = @card.original_text
+        = simple_form_for @card, url: "/check", method: "post"  do |f|
+          .form-group
+            = f.input :original_text, as: :hidden
+            = f.input :translated_text, label: "Перевод", input_html: { value: "" }
+          .form-group
+            = f.button :submit, value: "Проверить"
+        - if flash[:notice] or flash[:error]
+          %div.alert{class: flash[:notice] ? "alert-success" : "alert-danger"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>`%div.alert` can be written as `.alert` since `%div` is implicit